.github/workflows/CI.yml

```diff
@@ -13,9 +13,21 @@ jobs:
   unitTests:
     runs-on: ubuntu-18.04
 
-    steps:
+    steps:  
       - name: Checkout code
         uses: actions/checkout@v1
+      - name: Cache .ivy2
+        uses: actions/cache@v1
+        with:
+          path: /home/runner/.ivy2
+          key: cmwell-unitTests-ivy2-${{ hashFiles('**/build.sbt') }}
+          restore-keys: cmwell-unitTests-ivy2-
```

Comment:
According to their docs when there are multiple restore keys:
1. If there is an exact match - take the key. In your case there won't be an exact match (the key itself always includes the hash)
2. On partial match (by prefix, I think): take into account **all** the partially matched restored keys and take the most recent one.
Using a prefix like `cmwell-unitTests-ivy2-` actually "kills" the ability to use cache from integration tests. Using `cmwell-ivy2-unitTests-` (in key and in restore keys) will allow it. Do you want it?
3. How are you going to solve the mac issue what its PR will be merged (if at all)?

Reply:
Thought about it again: you can use `cmwell-unitTests-ivy2-` and add another restore keys (that will point to the other job)